new/LIT6950Egziabeher.xml

```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6950Egziabeher" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እግዚአብሔር፡ ጸውአ፡ እምነ፡ ገሊላ፡ አንስተ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">ʾƎgziʾabǝḥer ṣawwǝʿa ʾǝmǝnna galilā ʾanǝsta... (Qǝne of ʿǝṭāna mogar type)</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>This qǝne poem is marked as being of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign> 
+                        in <ref type="mss" corresp="DMGYi135IIIC7"/> and in the edition.
+                </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
```

Comment:
PoemQene?

Reply:
Could you add the keyword "Qene" to the online form (or PoemQene, if it is decided to use this one as the genre keyword as well)? It is absent from there, though it is present in the schema in the right place, I believe
![image](https://github.com/BetaMasaheft/Works/assets/19650382/aea97727-f7f6-44bc-ae1b-801e1309ca07)